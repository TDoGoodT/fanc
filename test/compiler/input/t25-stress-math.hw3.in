void main() {
    int y = (45   - 17 b);
    printi(y);
	int z = (y / 55 b);
	printi(z);
	int x =  (13 / z);
	printi(x);
}
