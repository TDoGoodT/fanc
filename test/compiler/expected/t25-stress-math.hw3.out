28
0
Error division by zero
